terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
/var/spool/torque/mom_priv/jobs/1532880.polyp1.ie.lehigh.edu.SC: line 4: 19099 Aborted                 (core dumped) ../Release/vws ${INPUT_FOLDER} ${OUTPUT_FILE} ${MAX_TIME} ${GENERATOR} ${TERMS} ${MIP_SOLVER} ${PROVIDE_PRIMAL_BOUND}
